packages/@tinacms/schema-tools/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@tinacms/schema-tools",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "license": "Apache-2.0",
+  "buildConfig": {
+    "entryPoints": [
+      {
+        "name": "src/index.ts",
+        "target": "node"
+      }
+    ]
+  },
+  "devDependencies": {
+    "@tinacms/scripts": "workspace:*",
+    "@types/yup": "^0.29.10",
+    "jest": "^27.0.6",
+    "react": "16.14.0",
+    "typescript": "^4.3.5"
+  },
+  "peerDependencies": {
+    "react": ">=16.14.0",
```

Comment:
I wouldn't think we need this as a peer dep or devDep, I don't see any usage of React in here and if/when we use this package inside `@tinacms/graphql` it won't meet this requirement.

Reply:
It is only needed for the `FC` type

Reply:
Ok, when we fixup our types we can probably keep this over in `tinacms` and have the `ui` portion of these types be inserted via generic.